terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -106,4 +108,19 @@ public ApServerName getApServerName() {
     public String getApServerVersion() {
         return webDriver.findElement(By.id("apServerVersion")).getText();
     }
+
+    /**
+     * Get the specified link.
+     * @param text for Identifying to look for the link
+     * @return link matching the text
+     */
+    public WebElement getTextMatchLink(String compareText) {
+        List<WebElement> elements = webDriver.findElements(By.tagName("a"));
+        for (WebElement element : elements) {
+            if (element.getText().equals(compareText)) {
+                return element;
+            }
+        }
+        throw new NoSuchElementException(compareText + "is not found.");
+    }
```

Comment:
Doesn't it get even if implemented like [PagenationTest](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/master/terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/pagination/PaginationTest.java#L56)?

Reply:
When use Xpath or linkText, selenium returns `NoSuchElementException` because the text on browser contains blanks and linefeed code.
This Error occurs only at `CodeListTest` due to the target text containing blanks.